backend/src/test/java/com/timlenny/backend/service/StudyplanServiceTest.java

```diff
@@ -23,7 +23,7 @@ class StudyplanServiceTest {
 
     @Test
     void isGetAllTopicsReturningAllTopics_whenGetAllTopicsCalled() {
```

Comment:
What if the repo is empty? Have you tested for this case?

Reply:
Is now implemented